src/common/styled-button.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import injectSheet, { StyleSheet } from 'react-jss';
+import { Button } from '@material-ui/core';
+
+const styles: StyleSheet = { 
+  button: {
+    backgroundColor: '#435160',
+    color: '#FFFFFF',
+    '&:hover': {
+      backgroundColor: '#3c4956'
+    }
+  },
+};
+
+export type ButtonProps = {
+  onClick?: ((event: React.MouseEvent<HTMLElement>) => void)  
+}
+
+
+export const StyledButton = injectSheet(styles)<ButtonProps>(({classes, children, onClick}) => (
+  <Button variant="contained" size='large' className={classes.button} onClick={onClick}>{children}</Button>
```

Comment:
should variant and size be passed in props to styled button ?

Reply:
@sk91 yeah, it could but I didn't find the need for that now as all the buttons are large on the wallet.